# robot_state_log_plotter.py config file

robot_state_log_plotter:

  actuators_config_path: $(find awesome_leg)/config/actuators.yaml

  urdf_path: $(find awesome_leg)/description/urdf/generated/awesome_leg_reduced_calibrated.urdf # urdf relative path (wrt to the package)
  srdf_path: $(find awesome_leg)/description/srdf/awesome_leg_reduced.srdf # srdf relative path (wrt to the package)
  
  # matfile_path: $(find awesome_leg)/$(find awesome_leg)//fixed_hip_setup/tip_cart_acc_cntrl/checks/real_tests/1_04_22/robot_state_log1.mat # logged data mat file path
  matfile_path: $(find awesome_leg)/test_results/sliding_hip_setup/jump_test_28_07_22/robot_state_logger/robot_state_log__0_2022_07_28__17_50_26.mat

  save_fig: false # whether or not to save figures to the destination folder (defaults to true if show_plots is false)
  
  save_fig_path: /tmp